.bazelrc

```diff
@@ -118,37 +131,26 @@ build:rs --bes_results_url="https://source.cloud.google.com/results/invocations/
 build:rs --project_id=iree-oss
 
 ###############################################################################
+# Windows specific flags for building with VC.
```

Comment:
This is awesome :D

I tested locally (threadripper 3970x so unfair comparison with other CPUs).

My builds before this were ~1h, with nearly all of the time spent compiling a long tail of TF kernels on a few threads. The build would run in parallel efficiently for ~10 minutes then be basically idle working on ~8 kernel files for 50 minutes.

Test steps (using Colab as the target since it pulls in most of the relevant deps):
```
bazel clean --expunge
python3 configure_bazel.py
python3 .\colab\start_colab_kernel.py
```

user.bazelrc (note that I use separate caches and output_user_root for separate repositories/branches to try to keep as much state cached per branch as possible)
```
startup --output_user_root=D:/dev/cache/bazeluserroot
build --disk_cache=D:/dev/cache/bazelcache --experimental_guard_against_concurrent_changes
build:debug --compilation_mode=dbg --copt=/O2 --per_file_copt=iree@/Od --strip=never
build --define=iree_tensorflow=true
```

Timing:

```
8:58 start
  300s "Fetching @aws;"
9:03 finished fetching, build running (128 actions in parallel)
9:04 821/2656
9:05 3747/4793 (working on tf targets)
9:06 4005/4793 (tf kernels, core/util/batch_util.cc)
9:07 4885/5342 (starting llvm/lib/CodeGen/...)
9:08 5275/5726 (more LLVM)
9:09 5893/6256 (LLVM, MLIR, IREE)
9:10 6693/6842 (IREE, XLA, MLIR, linking IREE python bindings)
finished 9:10
```

Reply:
(super scientific :P looks like https://docs.bazel.build/versions/master/skylark/performance.html#profile-and-analyze-profile would give more concrete data)